### PR TITLE
test: Created a new GitHub action to reuse the creation of a KFP cluster

### PR DESCRIPTION
**Pull request by hbelmiro**
**Description of your changes:**
This PR adds a new GitHub action to be reused by test workflows. This is intended to avoid code duplication across multiple workflows.

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

**Comment by hbelmiro** [comment]
cc @chensun

**Review by DharmitD — COMMENTED** [review]
/lgtm

**Comment by rimolive** [comment]
cc @james-jwu

**Comment by hbelmiro** [comment]
cc @chensun @zijianjoy @james-jwu

File: .github/actions/kfp-cluster/action.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        version: v0.22.0
+        node_image: kindest/node:v1.29.2
+
+    - name: Build images
+      shell: bash
+      run: ./scripts/deploy/github/build-images.sh
+
+    - name: Deploy KFP
+      shell: bash
+      run: ./scripts/deploy/github/deploy-kfp.sh
```
Comment (DharmitD) [inline .github/actions/kfp-cluster/action.yml:21]: @hbelmiro looks like this script deploys just kfp-tekton and not pipelines. @rimolive could you confirm?

Comment (hbelmiro) [reply .github/actions/kfp-cluster/action.yml:21]: Fixed.

**Review by DharmitD — COMMENTED** [review]
/lgtm

cc: @rimolive @chensun

**Pull request merged**